.circleci/config.yml

```diff
@@ -25,7 +25,7 @@ workflows:
       # parameters to update, i.e. which builds to run.
       - path-filtering/filter:
           name: check-updated-files
-          base-revision: main
+          base-revision: develop
```

Comment:
Shouldn't this stay to `main`?

Reply:
Yeah that's true. For the moment it doesn't matter cause main & develop will be in sync, but will revert it when I do a PR against main for final cleanup.